examples/dataresidency/set_region.py

```diff
@@ -0,0 +1,54 @@
+import sendgrid
+import os
+
+from sendgrid import Email, To, Content, Mail
+
+# Example 1
+# setting region to be "global"
+
+sg = sendgrid.SendGridAPIClient(api_key=os.environ.get('SENDGRID_API_KEY'))
+sg.set_data_residency("global")
+from_email = Email("example@abc.com")
+to_email = To("example@abc.com")
+subject = "Sending with SendGrid is Fun"
+content = Content("text/plain", "and easy to do anywhere, even with Python")
+print(sg.client.host)
+mail = Mail(from_email, to_email, subject, content)
+response = sg.client.mail.send.post(request_body=mail.get())
+print(response)
+print(response.status_code)
+print(response.body)
+print(response.headers)
+
+
+# Example 2
+# setting host
+sg = sendgrid.SendGridAPIClient(api_key=os.environ.get('SENDGRID_API_KEY'))
+sg.set_host("https://api.eu.sendgrid.com")
```

Comment:
should we add a comment noting that "set_host" and "set_data_residency" are mutually exclusive ways to do the same thing?

Reply:
We can add the comment